README.md

```diff
@@ -7,6 +7,8 @@ API publishing, data access, and a local pub/sub event system.
 
     glide install --strip-vendor
```

Comment:
IMO, the best solution (you were also suggesting in the same direction I guess) is to
 (1) Make sure we update glide.lock (or glide.yaml?) with the commit Id 
(2) The makefile has ldflags for each of the plugins, which it fills from glide.lock at compile time
(3) At run time, will print the hash for each of the plugin
(4) If not filled, it will be empty

Note: This puts the onus back on the developer to update glide.lock. If he/she forgets to update the glide.lock we will be back in the same situation as before.